src/etc/inc/interfaces.inc

```diff
@@ -3044,9 +3044,9 @@ function interface_dhcpv6_prepare($interface = 'wan', $wancfg)
     chmod("/var/etc/dhcp6c_{$interface}_script.sh", 0755);
 
     $dhcp6ccommand = exec_safe(
-        "/usr/local/sbin/dhcp6c %s -c %s -p %s %s",
+        "/usr/local/sbin/dhcp6c {$wancfg['adv_dhcp6_debug']} %s -c %s -p %s %s",
```

Comment:
this breaks security by feeding arbitrary values to the command line without escaping

Reply:
Yes, that's why I commented I could not get the array way to work, my php skills are limited to 'hello world'

Reply:
It's a bit screwed then, because with the existing code it's either debug low level or debug high level. you cannot turn off the debug completely because of the '-' at the start if the array string, if both debug and no release are empty then the command fails.